### PR TITLE
PP-9288: Update alpine to 3.15

### PR DESCRIPTION
**Pull request by oswaldquek**
Manage node version via the .nvmrc file and specify it in the package.json too.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+12.22.7
```
Comment (jfharden) [inline .nvmrc:1]: I think you mean 12.22.10 right?

**Comment by jfharden** [comment]
You need to update the github unit test workflow too: https://github.com/alphagov/pay-stream-s3-sqs/blob/master/.github/workflows/run-tests.yml

File: .nvmrc (hunk 1 above)
Comment (oswaldquek) [reply .nvmrc:1]: Argh I copied the file from selfservice but hadn't updated the codebase

Comment (oswaldquek) [reply .nvmrc]: fixed

**Comment by oswaldquek** [comment]
@jfharden Review comments addressed.

**Review by jfharden — APPROVED** [review]
Ace! Make sure once you have pushed the image to test it by running (in test) a toolbox "update transactions" job :)

**Pull request merged**